app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryLabels.js

```diff
@@ -60,7 +64,9 @@ class ExpandedStoryLabels extends React.Component {
 
 ExpandedStoryLabels.propTypes = {
   labels: PropTypes.arrayOf(PropTypes.object).isRequired,
-  onEdit: PropTypes.func.isRequired
+  onEdit: PropTypes.func.isRequired,
+  addLabel: PropTypes.func.isRequired,
+  projectLabels:PropTypes.arrayOf(PropTypes.object).isRequired
```

Comment:
As we are already using models, one cool thing you could do is, add a new `shape` function to the `Label` model, and pass it here, something like:

```js
export const shape = () => PropTypes.shape({
  someField: PropTypes.string,
  otherField: PropTypes.number,
})

ExpandedStoryLabels.propTypes = {
  projectLabels:PropTypes.arrayOf(Label.shape()).isRequired
}
```

What do you think? cc @talyssonoc

Reply:
I don't like the idea of having proptypes directly in the model files but I do agree about having shapes for the models, maybe in a separate file/folder. We could do that in a separate PR and then begin to add to every new PR, what do you think? Or we could start in this one, it's up to you @kostadriano

Reply:
I think a separate PR is better, this way we can change the other components as well to keep the pattern. Thanks for the idea @iagodahlem